### DISCUSSION TITLE
Vendor Product availability update via SP-API (replace EDI COSTINV)

### DISCUSSION THREAD

**Discussion by MktpStar** [amzn/selling-partner-api-models#3071] [General] [answered]
Hi,
I've been trying for more than a week to find a way to update the product stock/availability fro my Vendor account with no succes.
Patch to '/attributes/fulfillment_availability' complains that fulfillment_availability is invalid.
Tried with JSON_LISTINGS_FEED as well, no dice.
Tried to get some help from Vendor Central, nothing conclusive yet.
So I'm turning to you guys and girls: what's the best way to update product availability/stock for a Vendor ("sold by Amazon", not FBA).
Thank you in advance for your time!

**Comment by jlevers** [comment]
I'm also trying to figure this out. Any input on how to update Vendor product availability via the SP API would be much appreciated.

**Comment by MktpStar** [comment, marked as answer]
Hi Jesse,
Thank you for your hard work!
I've been down this road, sorry for not updating this thread.
unfortunatelly this is not possible with SP-API, only EDI.

In Vendor Central, in the top menu , under "Integration (EDI/API)", the first link "Getting started" explains which business processes are available in EDI and API (scroll down after the big images).

For Item Inventory there's nothing yet with API.
In the end, I switched back to COSTINV.
